developer-hub/src/reference_docs/glossary.rs

```diff
@@ -36,41 +36,81 @@
 //!
 //! #### Offchain
 //!
+//! Refers to operations conducted outside the blockchain's consensus mechanism. They are essential
+//! for enhancing scalability and efficiency, enabling activities like data fetching and computation
+//! without bloating the blockchain.
+//!
 //! #### Host Function:
 //!
+//! Host functions are functions provided by the runtime environment (the [host](#host)) to the Wasm
+//! runtime. These functions allow the Wasm code to interact with and perform operations on the
+//! [node](#node), like accessing the blockchain state.
+//!
 //! #### Runtime API:
 //!
+//! The runtime API acts as a communication bridge between the runtime and the node, serving as the
+//! exposed interface that facilitates their interactions.
+//!
 //! #### Dispatchable:
 //!
-//! Callable
+//! Dispatchables are functions that can be called by external entities, such as users or external
```

Comment:
i would say it is a function object, which can be called (internally or externally, probably not that important). it is more term to substrate. `functions that can be called by external entities` - closer to extrinsic.